app/views/articles/show.html.erb

```diff
@@ -165,7 +165,7 @@
               <% elsif should_show_crossposted_on?(@article) %>
                 <em>
                   Originally published at
-                  <a href="<%= @article.canonical_url || @article.feed_source_url %>" style="color:#1395b8"><%= get_host_without_www(@article.canonical_url || @article.feed_source_url) %></a>
+                  <a href="<%= @article.canonical_url || @article.feed_source_url %>" style="color:#1395b8" class="u-in-reply-to"><%= get_host_without_www(@article.canonical_url || @article.feed_source_url) %></a>
```

Comment:
The `u-in-reply-to` does not appear to exist anywhere in the codebase or this PR except for here. I didn't realize this was required for webmentions, https://indieweb.org/reply . 

Same goes for the other `u-*`, `h-entry` etc. CSS class names. Consider mentioning this in the PR description for those unfamiliar with webmentions.

Reply:
Thanks @nickytonline, I'll update the PR description